src/lib/modules/blockchain_process/blockchain.js

```diff
@@ -27,6 +28,13 @@ var Blockchain = function(userConfig, clientClass) {
   this.proxyIpc = null;
   this.isStandalone = userConfig.isStandalone;
 
+  if (this.userConfig.account) {
```

Comment:
Should this be in the config parser?
Arriving at this point I would expect the config to be valid